docker/dev_compose/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: "3.7"
+services:
+
+  dietstory_database:
+    image: mysql:5.6.45
+    hostname: dietstory_database
+    ports:
+      - "3306:3306"
+    networks:
+      - dietstory_db
+    command: --default-authentication-plugin=mysql_native_password
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD}
+
+  populate_database:
+    image: benjixd/dietstory-populate-db
+    networks:
+      - dietstory_db
+    volumes:
+      - "${DIETSTORY_PATH}:/mnt"
+    environment:
+      MYSQL_HOST_NAME: dietstory_database
+      MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD}
+    restart: on-failure
+    depends_on:
+      - dietstory_database
+
+  dietstory_server:
+    image: benjixd/dietstory
+    ports:
+      - "7575:7575" 
+      - "7576:7576" 
+      - "7577:7577"
+      - "8484:8484"
+    networks:
+      - dietstory_traffic
+      - dietstory_db
+    volumes:
+      - "${DIETSTORY_PATH}:/mnt"
+    environment:
+      MYSQL_HOST_NAME: dietstory_database
+      MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD}
+    restart: on-failure
+    depends_on:
+      - populate_database
+
+networks:
+  dietstory_db:
+  dietstory_traffic:
+
```

Comment:
Docker-compose file spinning up MySQL:5.6.45, Dietstory and a mysqld pushing transactions to MySQL DB.

Reply:
Consider adding an environment variable for a path on the host machine. Mount this path to /var/lib/mysql of the database container. This will ensure the database writes to the host machine, preventing database wipe every time we run `docker-compose down`

Reply:
Docker-compose now has a `volume` field for dietstory_database. The change requires you to mount a folder for your DB, but the populate_db container will only run if the schema doesn't exist.